### ISSUE TITLE
Misalignment of org-table caused by targets and links

### ISSUE THREAD

**Issue by ujikol** [kunalb/poet#22]
![image](https://user-images.githubusercontent.com/5747033/107660163-41c95e00-6c88-11eb-9828-6ccf24f5842a.png)

**Comment by ujikol** [comment]
Can you give me quick solution/work-around?
`(face-remap-add-relative 'org-link :fixed-pitch t)` does not work

**Comment by kunalb** [comment]
The quickest one would be to make links fixed-pitch everywhere by customizing the face; I don't have a clean solution for just tables at the moment.

**Comment by ujikol** [comment]
Yes, but how can I change a face exactly? `(face-remap-add-relative 'org-link :fixed-pitch t)` does not work.

**Comment by kunalb** [comment]
Oh, sorry I didn't understand -- this should do it (after loading the theme).
```
(set-face-attribute 'org-target nil 
                    :inherit 'fixed-pitch)
(set-face-attribute 'org-link nil 
                    :foreground "#0000ff"
                    :inherit 'fixed-pitch)
```
